draft-ietf-quic-transport.md

```diff
@@ -3751,13 +3793,15 @@ been lost.  In general, information is sent again when a packet containing that
 information is determined to be lost and sending ceases when a packet
 containing that information is acknowledged.
 
-* Data sent in CRYPTO frames are retransmitted according to the rules in
-  {{QUIC-RECOVERY}}, until either all data has been acknowledged or the crypto
-  state machine implicitly knows that the peer received the data.
+* Data sent in CRYPTO frames is retransmitted if detected as lost, e.g.
+  according to the rules in section 4.3.1 of {{QUIC-RECOVERY}}, until either all
+  data has been acknowledged or the crypto state machine implicitly knows that the
+  peer received the data.
 
-* Application data sent in STREAM frames is retransmitted in new STREAM frames
-  unless the endpoint has sent a RST_STREAM for that stream.  Once an endpoint
-  sends a RST_STREAM frame, no further STREAM frames are needed.
+* Application data sent in STREAM frames is retransmitted if detected as lost,
+  e.g. according to the rules in {{QUIC-RECOVERY}}, in new STREAM frames unless
```

Comment:
Having "in new STREAM frames" after the comma reads oddly.  Perhaps this needs to become its own sentence?  "New STREAM frames containing the lost data are generated, unless the endpoint has sent a RST_STREAM for that stream."

Reply:
done